src/wazuh_modules/vulnerability_scanner/schemas/scanVulnerabilityCandidate.fbs

```diff
@@ -0,0 +1,23 @@
+namespace NSVulnerabilityScanner;
```

Comment:
Probably, we can use all the same namespace to avoid multiple `using...`

Reply:
The issue I found when I tried to compile my fbs was that the namespace had the same name that the table. So, later in the code compilation they had names miss understanding.
At this point I just added "NS" at the start of the namespace just to solve that compilation naming problem.
But, if we all use the same namespace for me is a right choice.

Reply:
We agree to use all the same namespace.